nav2_bt_waypoint_follower/src/ros_topic_logger.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <utility>
+#include "nav2_bt_waypoint_follower/ros_topic_logger.hpp"
+#include "tf2_ros/buffer_interface.h"
+
+namespace nav2_bt_navigator
```

Comment:
This file looks like a complete copy paste, why not use the bt navigator version of this object?

Reply:
bt_navigator is not exporting this.
Based on https://github.com/ros-planning/navigation2/pull/1796#discussion_r439714103,
> If this demo requires changes in bt_navigator, maybe you could do it in different and concrete PRs, in which we could discuss its convenience, but not linked to a specific demo.

it is needed to avoid making changes to bt_navigator.

Reply:
If it's ok to change bt_navigator, I'll change `bt_navigator/CMakeLists.txt` and reuse the code.

Reply:
Or should I create another PR?

Reply:
Removed codes that was using RosTopicLogger. We'll add it if we ever need it.